website/content/docs/other-specifications/volume/index.mdx

```diff
@@ -144,7 +144,8 @@ parameters {
   of strings passed directly to the CSI plugin to validate the volume. The
   details of these parameters are specific to each storage provider, so consult
   the specific plugin documentation for more information. Only allowed on
-  **volume registration**.
+  **volume registration**. Note that this block is declarative, and an update 
+  replaces it in its entirety, thefore all paramters need to be specified.
```

Comment:
The wording here makes it sound a little bit like this is an exception. Maybe something like:

```suggestion
  **volume registration**. Note that, like the rest of the volume specification, 
  this block is declarative, and an update replaces it in its entirety, therefore 
  all parameters need to be specified.
```

(Also fixes spelling errors)